contrib/babelfishpg_tds/src/backend/tds/tdsprinttup.c

```diff
@@ -60,6 +60,13 @@ TdsPrinttupStartup(DestReceiver *self, int operation, TupleDesc typeinfo)
 {
 	DR_printtup *myState = (DR_printtup *) self;
 	Portal		portal = myState->portal;
+	PlannedStmt *plannedStmt = PortalGetPrimaryStmt(portal);
+	List 		*targetList = NIL;
+
+	if (portal->strategy != PORTAL_MULTI_QUERY)
```

Comment:
can TdsSendRowDescription handle NULL targetList?

Reply:
This is taken from FetchPortalTargetList where it could return NIL.